framework/src/utils/ColumnMajorMatrix.C

```diff
@@ -265,5 +266,5 @@ ColumnMajorMatrix::inverse(ColumnMajorMatrix & invA) const
 #endif
 
   if (return_value)
-    mooseError("error in lapack inverse solve");
+    throw MooseException(return_value);
```

Comment:
We really need to make `mooseException()` quickly before we get to many of these in the code!

@wilkandy : this looks good for now though